pandas/core/arrays/masked.py

```diff
@@ -1311,18 +1311,18 @@ def map(self, mapper, na_action=None):
     @overload
     def any(
         self, *, skipna: Literal[True] = ..., axis: AxisInt | None = ..., **kwargs
-    ) -> bool:
+    ) -> np.bool_:
```

Comment:
The pandas code expects it to return np.bool_ and not bool (see failing CI on last commit)!

Reply:
I think it would be better longer term if this returned `bool`

Reply:
It is easy to fix inside all/any but there are at least two places (maybe more) that error after that change:
pandas/core/arrays/masked.py:1164
pandas/core/arrays/masked.py:840

Reply:
OK this can be a follow up